src/setupSessionId.ts

```diff
@@ -0,0 +1,25 @@
+import { ulid } from 'ulidx';
+
+export const SESSION_ID_KEY = 'yext_analytics_session_id';
```

Comment:
Do we need to export the SESSION_ID_KEY? I'm not seeing it being used anywhere else.

Reply:
Removed `export`